tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27}-django{18,111}
+
+# The ordering of deps and commands here is weird due to the way we install xblock-sdk and associated dependencies.
+# Installing test/base.txt inside deps fails, and installing the env-specific Django versions inside deps
+# causes Django 1.4 to be installed over it by something later in the process.
+[testenv]
+passenv = CI TRAVIS TRAVIS_* DISPLAY
+deps =
+    -rrequirements.txt
+    -rtest_requirements.txt
+    -egit+https://github.com/edx/xblock-sdk.git#egg=xblock-sdk
+commands =
+    pip install -r {envdir}/src/xblock-sdk/requirements/test.txt
+    pip install -r {envdir}/src/xblock-sdk/requirements/base.txt
+    django18: pip install Django>=1.8,<1.9
+    django110: pip install Django>=1.10,<1.11
+    django111: pip install Django>=1.11,<2.0
+    pep8 xblockutils --max-line-length=120
+    pylint xblockutils
```

Comment:
Maybe split pep8 and pylint into a separate tox env so they don't get run twice?  See https://github.com/edx/django-user-tasks/blob/master/tox.ini#L44 and https://github.com/edx/django-user-tasks/blob/master/.travis.yml#L14 for an example

Reply:
Ah good catch. I've done that elsewhere, but missed it here.